compose.yml

```
@@ -0,0 +1,27 @@
+services:
+  app:
+    ports:
+      - "8086:80"
+    depends_on:
+      - db
```

Il manque des instructions pour que le compose.yaml ait une chance de fonctionner.

```suggestion
    env_file:
      - .env
    build: ./docker/php
    ports:
      - 8086:80
    depends_on:
      - db
    volumes:
      - ./:/var/www/html
```